crates/breez-sdk/lnurl/src/main.rs

```diff
@@ -0,0 +1,183 @@
+use std::{path::PathBuf, sync::Arc};
+
+use anyhow::anyhow;
+use axum::{
+    Extension, Router,
+    extract::DefaultBodyLimit,
+    http::{self, Method},
+    routing::{delete, get, post},
+};
+use clap::Parser;
+use diesel::{
+    SqliteConnection,
+    r2d2::{ConnectionManager, Pool},
+};
+use figment::{
+    Figment,
+    providers::{Env, Format, Serialized, Toml},
+};
+use serde::{Deserialize, Serialize};
+use spark_wallet::{DefaultSigner, Network, SparkWalletConfig};
+use tower_http::cors::{Any, CorsLayer};
+use tracing::{error, info};
+use tracing_subscriber::{EnvFilter, layer::SubscriberExt, util::SubscriberInitExt};
+
+use crate::state::State;
+
+mod models;
+mod routes;
+mod sqlite;
+mod state;
+
+#[derive(Clone, Parser, Debug, Serialize, Deserialize)]
+#[command(version, about, long_about = None)]
+struct Args {
+    /// Address the lnurl server will listen on.
+    #[arg(long, default_value = "127.0.0.1:8080")]
+    pub address: core::net::SocketAddr,
+
+    #[arg(long, default_value = "lnurl.conf")]
+    pub config: PathBuf,
+
+    /// Automatically apply migrations to the database.
+    #[arg(long)]
+    pub auto_migrate: bool,
+
+    /// Connectionstring to the postgres database.
+    #[arg(long, default_value = "")]
+    pub db_url: String,
+
+    /// Loglevel to use. Can be used to filter loges through the env filter
+    /// format.
+    #[arg(long, default_value = "info")]
+    pub log_level: String,
+
+    /// Spark network.
+    #[arg(long, default_value = "mainnet")]
+    pub network: Network,
+
+    /// Scheme prefix for lnurl urls.
+    #[arg(long, default_value = "https")]
+    pub scheme: String,
+
+    /// Domain for lnurl urls.
+    #[arg(long, default_value = "127.0.0.1:8080")]
+    pub domain: String,
+
+    /// Minimum amount that can be sent in a lnurl payment.
+    #[arg(long, default_value = "1000")]
```

Comment:
I think we can lower here to 1 sat as default?

Reply:
Those are actually millisatoshi. The 4000000 one should be bumped.

Reply:
I wouldn't do that ATM.